source: https://git.archlinux.org/svntogit/packages.git/tree/trunk/0001-build-disable-libnm-glib-support-by-default.patch?h=packages/networkmanager-openvpn

From 52aad4911ffad04cf8274b5574124a4d5470c7fa Mon Sep 17 00:00:00 2001
Message-Id: <52aad4911ffad04cf8274b5574124a4d5470c7fa.1549642989.git.jan.steffens@gmail.com>
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Wed, 17 Oct 2018 15:21:30 +0200
Subject: [PATCH] build: disable libnm-glib support by default

By now nobody should be using this. Keep the code around for a little
longer just in case anybody still uses this.

The libnm-glib support also serves as an example how do we build two
different versions of the properties plugin. We'll soon be in a similar
situation with Gtk 4.0. (sigh.) Just don't drop it yet.

Conflicts:
	NEWS
---
 configure.ac | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 11ff5d0..eaf015b 100644
--- configure.ac
+++ configure.ac
@@ -75,20 +75,20 @@ dnl
 dnl GNOME support
 dnl
 AC_ARG_WITH(gnome, AS_HELP_STRING([--without-gnome], [Build NetworkManager-openvpn without GNOME support, e.g. vpn service only]), [], [with_gnome_specified=no])
-AC_ARG_WITH(libnm-glib, AS_HELP_STRING([--without-libnm-glib], [Build NetworkManager-openvpn without libnm-glib comatibility]), [], [with_libnm_glib_specified=no])
+AC_ARG_WITH(libnm-glib, AS_HELP_STRING([--with-libnm-glib], [Build NetworkManager-openvpn with libnm-glib comatibility (depreacted)]), [], [with_libnm_glib_specified=no])
 if test "$with_libnm_glib_specified" != no -a "$with_libnm_glib" != no; then
 	if test "$with_gnome_specified" != no -a "$with_gnome" == no; then
 		AC_MSG_ERROR(Building --with-libnm-glib conflicts with --without-gnome)
 	fi
 fi
 if test "$with_gnome" != no; then
 	with_gnome=yes
 fi
 if test "$with_libnm_glib_specified" == no; then
-	with_libnm_glib="$with_gnome"
+	with_libnm_glib=no
 fi
-if test "$with_libnm_glib" != no; then
-	with_libnm_glib=yes
+if test "$with_libnm_glib" != yes; then
+	with_libnm_glib=no
 fi
 AM_CONDITIONAL(WITH_GNOME, test "$with_gnome" != no)
 AM_CONDITIONAL(WITH_LIBNM_GLIB, test "$with_libnm_glib" != no)
-- 
2.20.1
